lib/drivers/bayerContourNext.js

```diff
@@ -403,6 +403,7 @@ module.exports = function (config) {
     var listenTimer = setInterval(function () {
       hidDevice.receive(function(raw) {
         var packet = new Uint8Array(raw);
+        clearTimeout(abortTimer);
```

Comment:
Don't you need to reset the timeout here, rather than just clear it? What happens if the interval keeps firing, but never reaches a clearInterval? The interval would keep going forever, because the timeout was cancelled at the first interval.